control/tap-ctl-ipc.c

```diff
@@ -187,6 +187,13 @@ tap_ctl_connect(const char *name, int *sfd)
 
 	memset(&saddr, 0, sizeof(saddr));
 	saddr.sun_family = AF_UNIX;
+
+	if (unlikely(strlen(name) > sizeof(saddr.sun_path) - 1)) {
```

Comment:
I agree changing 'sizeof(saddr.sun_path) - 1" to '(sizeof(saddr.sun_path) - 1)' would make it a bit more readable